terraform/aks/databases.tf

```diff
@@ -0,0 +1,18 @@
+module "postgres" {
+  source = "git::https://github.com/DFE-Digital/terraform-modules.git//aks/postgres?ref=testing"
+
+  namespace             = var.namespace
+  environment           = local.environment
+  azure_resource_prefix = var.azure_resource_prefix
+  service_name          = local.service_name
+  service_short         = var.service_short
+  config_short          = var.config_short
+
+  cluster_configuration_map = module.cluster_data.configuration_map
+
+  use_azure               = var.deploy_azure_backing_services
+  azure_enable_monitoring = var.enable_monitoring
+  azure_extensions        = ["pg_stat_statements", "pgcrypto"]
```

Comment:
Add 'plpg' postgres extension

Reply:
"pg_stat_statements", "pgcrypto"  can be removed.

Reply:
Replaced with plpgsql   https://learn.microsoft.com/en-us/azure/azure-arc/data/using-extensions-in-postgresql-server